ksql-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java

```diff
@@ -290,6 +292,15 @@
               Optional.empty(),
               "Ensures legacy queries that perform a 'GROUP BY ROWKEY' continue to "
                   + "perform an unnecessary repartition step"
+          ),
+          new CompatibilityBreakingConfigDef(
```

Comment:
New breaking config added so that we can automagically add in the dummy `mapValues` step to maintain backwards compatibility.

Reply:
Yeah it is. Some, maybe all?, state stores in the latest version are now explicitly named.

Raised: https://github.com/confluentinc/ksql/issues/3656